File: 389. Find the Difference/Solution.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
 var findTheDifference = function(s, t) {
  let charCode = t.charCodeAt(t.length - 1)
  for (let i = 0; i < s.length; i++) {
      charCode -= s.charCodeAt(i)
      charCode += t.charCodeAt(i)
  }
  return String.fromCharCode(charCode)
};

File: 389. Find the Difference/Solution.java
class Solution {
  public char findTheDifference(String s, String t) {
      int charCode = (int) t.charAt(t.length() - 1);
      for (int i = 0; i < s.length(); i++) {
          charCode -= (int) s.charAt(i);
          charCode += (int) t.charAt(i);
      }
      return (char) charCode;
  }
}
